src/main/pg/rx.c

```diff
@@ -45,11 +45,9 @@
 #elif defined(USE_SERIALRX_SBUS)
 #define SERIALRX_PROVIDER SERIALRX_SBUS
 #elif defined(USE_SERIALRX_SPEKTRUM)
-#define SERIALRX_PROVIDER SERIALRX_SPEKTRUM
+#define SERIALRX_PROVIDER SERIALRX_SPEKTRUM2048
```

Comment:
Is SERIALRX_SPEKTRUM2048 good default?

Maybe:
```suggestion
# if defined(USE_TELEMETRY_SRXL)
# define SERIALRX_PROVIDER SERIALRX_SRXL
# else
# define SERIALRX_PROVIDER SERIALRX_SPEKTRUM2048
#endif
```

Reply:
We also have SRXL2 :)

Reply:
Thanks.